pom.xml

```diff
@@ -115,11 +92,36 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <sourceDirectories>src</sourceDirectories>
                     <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
                     <consoleOutput>true</consoleOutput>
                     <failsOnError>true</failsOnError>
                     <linkXRef>false</linkXRef>
+                    <sourceDirectories>src</sourceDirectories>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
```

Comment:
what is the purpose of moving this plugin down?

and why the version isn't specified?